run/system-package/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
This is redundant as `node_modules` is already ignored in a higher-level `.gitignore` file.
Can we remove this?

Reply:
Done